internal/provider/schema_cloudflare_access_application.go

```diff
@@ -53,6 +53,12 @@ func resourceCloudflareAccessApplicationSchema() map[string]*schema.Schema {
 			Type:     schema.TypeString,
 			Optional: true,
 			Default:  "24h",
+			DiffSuppressFunc: func(k, oldValue, newValue string, d *schema.ResourceData) bool {
+				// Suppress the diff if it's a bookmark app type. Bookmarks don't have a session duration
+				// field which always creates a diff because of the default '24h' value.
+				appType := d.Get("type").(string)
+				return appType == "bookmark"
```

Comment:
I think we need to have this as a conditional instead of the return value as if there is another appType, this is always going to return false for the calculated diff. the end return value should still compare the old and new values.